### PR TITLE
fix qmk configuration

### PR DESCRIPTION
**Pull request by worms19**

### CODE REVIEW

**Comment by worms19** [comment]
i made a fix for qmk version 1.1.2

File: qmk/keymap.c [hunk 1]
```diff
@@ -21,7 +21,7 @@ const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
 
     [_SYM] = LAYOUT_split_3x6_3(
         KC_NO, KC_EXLM, KC_AT, KC_HASH, KC_DLR, KC_PERC,    /* */   KC_CIRC, KC_AMPR, KC_ASTR, KC_LPRN, KC_RPRN, KC_NO,
-        KC_NO, KC_GRAVE, KC_NO, KC_NO, KC_NO, KC_NO,        /* */   KC_MINS, KC_EQUAL, KC_LBRACKET, KC_RBRACKET, KC_BSLASH, KC_NO,
+        KC_NO, KC_GRAVE, KC_NO, KC_NO, KC_NO, KC_NO,        /* */   KC_MINS, KC_EQUAL, KC_LBRC, KC_RBRC, KC_SLASH, KC_NO,
```
Comment (jporter-dev) [inline qmk/keymap.c:24]: `KC_SLASH` should be `KC_BSLS` as it is replacing `KC_BSLASH`

File: qmk/config.h [hunk 2]
```diff
@@ -37,7 +37,7 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #define TAPPING_FORCE_HOLD_PER_KEY
 #define TAPPING_TERM_PER_KEY
 #define TAPPING_TERM 200
-#define IGNORE_MOD_TAP_INTERRUPT
```
Comment (jporter-dev) [inline qmk/config.h:40]: Why was this removed? Is there a replacement for `IGNORE_MOD_TAP_INTERRUPT` in the new version?

**Review by jporter-dev — CHANGES_REQUESTED** [review]
Thanks for the PR! I added a couple of comments. I'll see if I can get QMK installed again and build it too.

**Comment by jporter-dev** [comment]
The files in this repo were rather old. Removed them and [this repo](https://github.com/jporter-dev/zmk-config) should be used and is linked in the README.

**Pull request closed without merging**